hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -4925,7 +4940,9 @@ public List<SnapshotDiffJob> listSnapshotDiffJobs(String volume,
                                                     String jobStatus,
                                                     boolean listAll)
       throws IOException {
-
+    ResolvedBucket resolvedBucket = this.resolveBucketLink(Pair.of(volume, bucket), false);
```

Comment:
Add the check in `cancelSnapshotDiff` as well.

Reply:
done